contact-us.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="CSS/contact-us style/form.css">
+    <title>Contact us</title>
+</head>
+<body>
+    <div id="contactLogo">
+        <img src="/Assets/images/home logo/company_logo.svg" alt="Company logo">
+        <p>Pet Care</p>
+    </div>
+    <div class="form__container">
+        <form id="contactUs">
+            <div class="form__inputGroup">
+                <input type="text" id="name" placeholder="Your Name" required>
+            </div>
+            <div class="form__inputGroup">
+                <input type="tel" id="number" placeholder="Phone No." required>
+            </div>
+            <div class="form__inputGroup">
+                <input type="email" id="email" placeholder="Email" required>
+            </div>
+            <div class="form__inputGroup">
+                <textarea id="message" rows="4" placeholder="Your Message"></textarea>
+            </div>
+            <div id="buttons">
+                <button type="submit">SUBMIT</button>
+                <button type="reset">RESET</button>

```

Comment:
In real life have you ever seen a reset button on any form your used? 
This button can be found only in old tutorials. No actual use for it

Reply:
True in modern sites you dont see them... i can remove it